packages/application-shell/src/components/project-suspended/project-suspended.tsx

```diff
@@ -9,22 +9,22 @@ type Props = {
   isTemporary?: boolean;
 };
 
+const getTitleMessage = (props: Props): MessageDescriptor => {
+  /* NOTE: Other suspension reasons are `Payment` and `Other` which are not handled yet */
+  if (props.isTemporary === true)
+    return messages.temporaryMaintenanceSuspensionMessage;
+
+  return messages.defaultSuspensionMessage;
+};
+
 const ProjectSuspended = (props: Props) => {
   const intl = useIntl();
+  const titleMessage = getTitleMessage(props);
 
   <MaintenancePageLayout
     imageSrc={ProjectSuspendedSVG}
-    title={(() => {
-      /* NOTE: Other suspension reasons are `Payment` and `Other` which are not handled yet */
-      if (props.isTemporary === true)
-        return (
-          <FormattedMessage
-            {...messages.temporaryMaintenanceSuspensionMessage}
-          />
-        );
-      return <FormattedMessage {...messages.defaultSuspensionMessage} />;
-    })()}
-    label={intl.formatMessage(messages.title)}
+    title={<FormattedMessage {...titleMessage} />}
```

Comment:
Why not?

```suggestion
    title={intl.formatMessage(titleMessage)}
```

Reply:
Could work. Didn't want to change too much. Let me try.

Reply:
Won't work as the message goes to `Text` which needs a React.Node as children.